harness/determined/keras/_tf_keras_trial.py

```diff
@@ -689,6 +694,25 @@ def _compute_validation_metrics(self) -> workload.Response:
 
         return {"num_inputs": num_inputs, "validation_metrics": metrics}
 
+    def _global_barrier(self) -> None:
```

Comment:
non-blocking: small preference for putting this on the LoopTrialController

Reply:
i put it here since only this class uses the barrier, but I can move it over